src/utils.jl

```diff
@@ -1214,3 +1214,23 @@ function mesh3d_triangles(x, y, z, cns)
     end
     return X, Y, Z
 end
+
+const _attrsymbolcache = Dict{Symbol, Dict{Symbol, Symbol}}()
+
+get_attr_symbol(letter::Symbol, keyword::String) = get_attr_symbol(letter, Symbol(keyword))
+                                                
+function get_attr_symbol(letter::Symbol, keyword::Symbol)
+    lt = if haskey(_attrsymbolcache, letter)
+        _attrsymbolcache[letter]
+    else
+        _attrsymbolcache[letter] = Dict{Symbol, Symbol}()
+    end
+
+    lk = if haskey(lt, keyword)
+        lt[keyword]
+    else
+        lt[keyword] = Symbol(letter, keyword)
+    end
+
+    return lk
+end
```

Comment:
```suggestion
function get_attr_symbol(letter::Symbol, keyword::Symbol)
    return _attrsymbolcache[letter][keyword]
end
```

Reply:
Good idea. I think my mindset was on avoiding unused symbols to not add to startup costs.